app/templates/gulpfile.babel.js

```diff
@@ -19,7 +19,7 @@ gulp.task('styles', () => {<% if (includeSass) { %>
     }).on('error', $.sass.logError))<% } else { %>
   return gulp.src('app/styles/*.css')
     .pipe($.sourcemaps.init())<% } %>
-    .pipe($.autoprefixer({browsers: ['last 1 version']}))
+    .pipe($.autoprefixer({browsers: ['> 1%', 'last 2 versions', 'Firefox ESR']}))
```

Comment:
I have set default browsers as it listed in [browserlist](https://github.com/ai/browserslist#browserslist-)